pkgs/development/tools/protoc-gen-rust/default.nix

```
@@ -0,0 +1,31 @@
+{ fetchCrate
+, lib
+, rustPlatform
+, protobuf
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rust-protobuf";
```

Looks like you also forgot this one :)

```suggestion
  pname = "protoc-gen-rust";
```